docs/src/modules/components/PickersPlayground.tsx

```diff
@@ -41,6 +43,34 @@ import { isDatePickerView } from '@mui/x-date-pickers/internals/utils/date-utils
 // eslint-disable-next-line no-restricted-imports
 import { isTimeView } from '@mui/x-date-pickers/internals/utils/time-utils';
 
+const ComponentSection = styled('div')(({ theme }) => ({
+  padding: theme.spacing(4),
+  display: 'flex',
+  flexDirection: 'column',
+  gap: theme.spacing(2),
+  overflowX: 'auto',
+  '& .MuiPickersLayout-root': {
+    borderRadius: 6,
+    border: '1px dashed',
+    borderColor:
+      theme.palette.mode === 'light'
+        ? theme.palette.grey[300]
+        : alpha(theme.palette.grey[500], 0.1),
+    ...(theme.palette.mode === 'dark' && {
+      backgroundColor: alpha(theme.palette.grey[900], 0.2),
+    }),
+  },
+}));
+
+const PropControlsSection = styled('div')(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  background: alpha(theme.palette.grey[50], 0.5),
+  ...(theme.palette.mode === 'dark' && {
+    backgroundColor: alpha(theme.palette.grey[900], 0.3),
+  }),
+}));
+
 function RadioGroupControl({
```

Comment:
@noraleonte + @LukasTy What do y'all think about turning this radio group into a toggle button? It seems fitting as it's a three-option control. I've tried to do it myself, but I was getting this error:

<img width="400" alt="Screenshot 2024-01-15 at 18 29 07" src="https://github.com/mui/mui-x/assets/67129314/4d303c7b-b831-4954-8c17-482db5df2e3f">

Reply:
@danilo-leal Your suggestion makes a lot of sense! 🙌 
I've pushed a working implementation with `ToggleButtonGroup`. 😉 
Please feel free to refine. 👍

Reply:
Woo, thank you! 🤝